shared/constants/unlock-folders.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+
+import HiddenString from '../util/hidden-string'
+import type {TypedAction} from '../constants/types/flux'
+import type {DeviceID} from '../constants/types/flow-types'
+
+export type Device = {
+  type: 'desktop' | 'mobile',
```

Comment:
those are the only 2 types, its in constants.go which isn't generated. we should just manually add this to our generated types in flow-types actually